### PR TITLE
Added Aboobaker Siddiq R to members

### LINKED ISSUE #5100: [Community] Member Profile: Aboobaker Siddiq R
**Current Behavior**

@aboobakersiddiqr63 has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @aboobakersiddiqr63 as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @aboobakersiddiqr63
- Twitter: 
- LinkedIn: [in/aboobakersiddiqr](https://www.linkedin.com/in/aboobakersiddiqr/)
- Layer5 Cloud: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by Harsha-10**
**Description**

This PR fixes #5100

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 3ac6887852d8a4fe93442c199a89762f170ee5e0 at: https://65464f0affce1415a368e635--layer5.netlify.app

**Comment by Harsha-10** [comment]
Please review @aboobakersiddiqr63

File: src/collections/members/Aboobaker-Siddiq-R/index.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+name: Aboobaker Siddiq R
+position: Contributor
+image_path: ./Aboobaker-Siddiq-R.webp
+github: aboobakersiddiqr63
+linkedin: aboobakersiddiqr
+layer5: 846d0933-57da-44ff-9ae8-098007e03cb3
+location: Chennai, India
+bio:  I'm Aboobaker Siddiq, a cloud engineer and full stack developer. My passion for open source technologies drives me to craft innovative solutions that thrive in the digital landscape. I'm dedicated to creating robust software and web applications that seamlessly blend cutting-edge functionality with exceptional user experiences.
+badges: 
+  - meshery
```
Comment (sudhanshutech) [inline src/collections/members/Aboobaker-Siddiq-R/index.mdx:11]: @Harsha-10 can you also add here doc badge. Aboobaker is consistent contributor there also

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 3c43d5fe58021db29ca0ea3e8f6eb707cf223ade at: https://65466fd55f288e22cb05f06f--layer5.netlify.app

**Review by sudhanshutech — APPROVED** [review]
LGTM

**Pull request merged**